dockers/docker-sonic-gnmi/gnmi-native.sh

```diff
@@ -71,7 +71,7 @@ else
 fi
 
 # Enable ZMQ for SmartSwitch
-LOCALHOST_SUBTYPE=`sonic-db-cli CONFIG_DB hget localhost "subtype"`
+LOCALHOST_SUBTYPE=`sonic-db-cli CONFIG_DB hget "DEVICE_METADATA|localhost" "subtype"`
 if [[ x"${LOCALHOST_SUBTYPE}" == x"SmartSwitch" ]]; then
     TELEMETRY_ARGS+=" -zmq_address=tcp://127.0.0.1:8100"
```

Comment:
Add a comment to explain we don't really use "127.0.0.1"?

Reply:
Fixed, change to use zmq_port